### PR TITLE
Some improvements for developing

### PR DESCRIPTION
**Pull request by lciolecki**
* improvements at base configs
* merge all development configs into one with models: demo, standalone and flink example
* rename example to demo
* split docker-compose to nussknacker container
* upgrade version some docker containers
* make other small fixes to reflect changes above

### CODE REVIEW

File: demo/docker/README.md [hunk 1]
```diff
@@ -6,13 +6,12 @@ Docker images are published on each commit. You can find them at https://hub.doc
...
-* Env variable NUSSKNACKER_VERSION has to point to docker tag with model.
-  You can set it in .env file. 
-    * Sample file (.env) is provided. It assumes that jar with model is located at docker image in /opt/nussknacker/model/exampleModel.jar.
-    * ./downloadSampleAssembly.sh ([version]) script is also provided. It can build sample model or downloaded released version. 
-    * You can change model by mount downloaded version in docker-composer.yml  
-* `docker-compose up -d` or `docker-compose -f docker-compose-file.yml up -d`:)
+* Env variable NUSSKNACKER_VERSION has to point to docker tag with model. You can set it in .env file.
+* You can change each components version like kafka / flink / etc.. by setting corresponding env at .env file.
+* Full env: `docker-compose -f docker-compose.yml -f docker-compose-env.yml up -d` // Full env tests
+* App env: `docker-compose -f docker-compose.yml up -d` // Only for launch app at standalone mode
```
[the remark this answers is not in the document]
Comment (lciolecki) [reply demo/docker/README.md]: Changed.

File: demo/docker/downloadSampleAssembly.sh [hunk 2]
```diff
@@ -1,18 +0,0 @@
-#!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (lciolecki) [reply demo/docker/downloadSampleAssembly.sh]: Please see: nussknacker-dist/universal/conf/docker-application.conf. There is:
```
flinkConfig {
  restUrl: "http://jobmanager:8081"
  queryableStateProxyUrl: "taskmanager:9069"
  parallelism: 4
  jobManagerTimeout: 1m
  **classpath: ["model/exampleModel.jar"]**
}
```

File: ui/server/develConf/generic/application.conf [hunk 3]
```diff
@@ -4,11 +4,6 @@ commentSettings: {
   link: "https://github.com/TouK/nussknacker/$1"
 }
```
[the remark this answers is not in the document]
Comment (lciolecki) [reply ui/server/develConf/generic/application.conf]: Reverted.

File: ui/buildServer.sh [hunk 4]
```diff
@@ -8,8 +8,6 @@ fi
 cd ..
```
Comment (mproch) [inline ui/buildServer.sh]: maybe we want to get rid of this if altogether?

File: nussknacker-dist/src/universal/bin/nussknacker-entrypoint.sh [hunk 5]
```diff
@@ -15,7 +15,7 @@ GROUP=${DAEMON_GROUP-${6-"daemon"}}
 mkdir -p ${STORAGE_DIR}/logs
 mkdir -p ${STORAGE_DIR}/db
 
-chown -R ${USER}:${GROUP} ${STORAGE_DIR}
+#chown -R ${USER}:${GROUP} ${STORAGE_DIR}
```
Comment (mproch) [inline nussknacker-dist/src/universal/bin/nussknacker-entrypoint.sh]: ?

File: ui/server/README.md [hunk 6]
```diff
@@ -1,19 +1,18 @@
...
-##sample endpoints
-
-###process list
-```http://localhost:8081/api/processes```
+# Running full env (for integration tests)
+* Go to docker/demo and run `docker-compose -f docker-compose-env.yml up -d` //runs full env with kafka / flink / etc..
+* Run nussknacker by IntelliJ or `./runServer.sh`
+ 
+# Access to service
+ Service should be available at http://localhsot:8080/api
```
Comment (mproch) [inline ui/server/README.md]: typo

File: ui/buildServer.sh (hunk 4 above)
Comment (lciolecki) [reply ui/buildServer.sh]: In future we can use it ;) I will leave it.

File: nussknacker-dist/src/universal/bin/nussknacker-entrypoint.sh (hunk 5 above)
Comment (lciolecki) [reply nussknacker-dist/src/universal/bin/nussknacker-entrypoint.sh]: There is a problem to chown mouted volume - STORAGE_DIR. Docker can't chown at this path.

File: ui/server/README.md (hunk 6 above)
Comment (lciolecki) [reply ui/server/README.md]: fixed

**Comment by lciolecki** [comment]
> So, to sum up:
> 
> * merge all development configs into one
> * rename example to demo
> * split docker-compose to nussknacker container and the rest
> * make other small fixes to reflect changes above
>   Did I miss sth?
>   I had some minor comments, but looks pretty good, thanks +1

**Pull request merged**